.github/workflows/build.yml

```diff
@@ -189,7 +189,7 @@ jobs:
           cd install
           NAME=llvm-mingw-$TAG-ucrt-macos-universal
           mv llvm-mingw $NAME
-          tar -Jcf ../$NAME.tar.xz $NAME
+          tar -Jcf ../$NAME.tar.xz --format=ustar $NAME
```

Comment:
Can you add a comment in the commit message, about the reasoning for this tar invocation to be different from the others?